pydarn/plotting/rtp.py

```diff
@@ -335,11 +340,21 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
                                      start_time=start_time,
                                      end_time=end_time,
                                      opt_beam_num=cls.dmap_data[0]['bmnum'])
-        if coord is Coord.SLANT_RANGE:
+                   
+        if coord is Coords.SLANT_RANGE:
             # Get rxrise from hardware files (consistent with RST)
             rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
                                     .hardware_info.rx_rise_time
             y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+        elif coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
+            rxrise = SuperDARNRadars.radars[cls.dmap_data[0]['stid']]\
+                                    .hardware_info.rx_rise_time
+            y = gate2slant(cls.dmap_data[0], y_max, rxrise=rxrise)
+            Re = 6371
+            y = Re*np.arcsin(np.sqrt((y**2/4)-(hgt**2))/Re)  
+            y0inx = np.min(np.where(np.isfinite(y))[0])
+            y = y[y0inx:]
+            z = z[:,y0inx:]
```

Comment:
I'd maybe consider a comment around here to reference the equation in Bills paper. And a comment for line 355 just explaining quickly what it is doing, it's beautifully compact, but not immediately obvious.

Would this also benefit from being inside the `utils/conversions.py` module so that it could be used elsewhere if needed? This will also fix the issue of not being able to set a `hgt` for the `rtps` in summary plots.